frank-flow/src/frontend/src/app/shared/services/flow-structure.service.ts

```diff
@@ -280,6 +278,44 @@ export class FlowStructureService {
     }
   }
 
+  deleteAttribute(
+    key: string,
+    attributeList: FlowNodeAttributes,
+    flowUpdate = false
+  ): void {
+    let hasAttribute = false;
+    Object.entries(attributeList).forEach(
+      ([attributeKey, attribute]: [string, FlowNodeAttribute]) => {
+        if (attributeKey === key) {
+          const newValue = ``;
+
+          this.checkAttributeForSpecialChars(attribute);
+
+          hasAttribute = true;
+          this.monacoEditorComponent?.applyEdit(
+            {
+              startLineNumber: attribute.line,
+              startColumn: attribute.startColumn,
+              endColumn: attribute.endColumn,
+              endLineNumber: attribute.line,
+            },
+            newValue,
+            flowUpdate
+          );
+        }
+      }
+    );
+  }
+
+  checkAttributeForSpecialChars(attribute: FlowNodeAttribute): void {
+    if (attribute.value.match(/</g)) {
+      attribute.startColumn -= 3;
+    }
+    if (attribute.value.match(/>/g)) {
+      attribute.startColumn -= 3;
+    }
+  }
```

Comment:
Checking only these characters is to specific. We should be using a function like html_special_chars().